File: first_page_module/src/main/java/com/mobile/first_page_module/view/IFirstPageView.java
package com.mobile.first_page_module.view;

import com.mobile.common_library.base.IBaseView;
import com.mobile.first_page_module.bean.FirstPageResponse;

import java.util.List;

/**
 * author    : Urasaki
 * e-mail    : Urasaki@qq.com
 * date      : 2020/3/7 15:44
 * introduce :
 */


public interface IFirstPageView extends IBaseView {

    void firstPageDataSuccess(List<FirstPageResponse.ResultData.JuheNewsBean> success);

    void firstPageDataError(String error);
}

File: module_home/src/main/java/com/phone/module_home/view/IFirstPageView.kt
package com.phone.module_home.view

import com.phone.library_common.base.IBaseView
import com.phone.library_common.bean.FirstPageResponse.ResultData.JuheNewsBean

/**
 * author    : Urasaki
 * e-mail    : 1164688204@qq.com
 * date      : 2020/3/7 15:44
 * introduce :
 */
interface IFirstPageView:IBaseView {

    fun firstPageDataSuccess(success: List<JuheNewsBean>)

    fun firstPageDataError(error: String)
}